src/e2e/step-definitions/persoon.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import { Then } from "@cucumber/cucumber";
+import { z } from "zod";
+import { CustomWorld } from "../support/worlds/world";
+import { worldUsers } from "../utils/schemes";
+
+const ONE_MINUTE_IN_MS = 60_000;
+const FIFTEEN_SECONDS_IN_MS = 15_000;
+
+Then(
+  "{string} sees the person",
+  { timeout: ONE_MINUTE_IN_MS },
+  async function (this: CustomWorld, user: z.infer<typeof worldUsers>) {
+    await this.expect(this.page.getByText(`999993896`)).toBeVisible({
```

Comment:
I think for maintainability maybe better to make a const out of values like `999993896`? Similar to how we do it in our integration tests:

```
   const val TEST_PERSON_HENDRIKA_JANSE_BSN = "999993896"
    const val TEST_PERSON_HENDRIKA_JANSE_EMAIL = "hendrika.janse@example.com"
    const val TEST_PERSON_HENDRIKA_JANSE_BIRTHDATE = "1965-01-01"
    const val TEST_PERSON_HENDRIKA_JANSE_GENDER = "V"
    const val TEST_PERSON_HENDRIKA_JANSE_FULLNAME = "Héndrika Janse"
    const val TEST_PERSON_HENDRIKA_JANSE_PHONE_NUMBER = "0612345678"
    const val TEST_PERSON_HENDRIKA_JANSE_PLACE_OF_RESIDENCE =
        "Street ¦ 38 & House ¦ 10, Baghdad, Park Al-Sadoum, Hay Al-Nidhal 103"
        ```